CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/results/AnalyticResult.java

```diff
@@ -42,14 +42,14 @@
     protected AnalyticController analyticController = null;
     protected final List<ResultListener<D>> resultListeners = new ArrayList<>();
 
-    public void setSelectionOnGraph(final List<D> results) {
+    public void setSelectionOnGraph(final Iterable<D> results) {
         final List<Integer> verticesToSelect = new ArrayList<>();
         final List<Integer> transactionsToSelect = new ArrayList<>();
-        results.forEach(result -> {
-            if (result.getElementType() == GraphElementType.VERTEX) {
-                verticesToSelect.add(result.getElementId());
-            } else if (result.getElementType() == GraphElementType.TRANSACTION) {
-                transactionsToSelect.add(result.getElementId());
+        results.forEach(analyticResult -> {
+            if (analyticResult.getElementType() == GraphElementType.VERTEX) {
+                verticesToSelect.add(analyticResult.getElementId());
+            } else if (analyticResult.getElementType() == GraphElementType.TRANSACTION) {
+                transactionsToSelect.add(analyticResult.getElementId());
             } else {
                 // Do nothing
             }
```

Comment:
Why do we have an empty else block?

Reply:
@OrionsGuardian I believe it may have been response to a sonar code smell previously (that since then we no longer check for)

Reply:
Can we remove it?
It seems unnecessary to me.